.github/workflows/schema-check.yml

```diff
@@ -9,13 +9,13 @@
 # occur so we want to proactively alert to it.
 #
 # **when?**
-# On pushes to `develop` and release branches. Manual runs are also enabled.
+# Only can be run manually
 name: Artifact Schema Check
 
 on:
-  pull_request:
-    types: [ opened, reopened, labeled, unlabeled, synchronize ]
-    paths-ignore: [ '.changes/**', '.github/**', 'tests/**', '**.md', '**.yml' ]
+  #  pull_request:
+  #    types: [ opened, reopened, labeled, unlabeled, synchronize ]
+  #    paths-ignore: [ '.changes/**', '.github/**', 'tests/**', '**.md', '**.yml' ]
 
   workflow_dispatch:
```

Comment:
Let's add a branch input here that defaults to main.  Then we could run it for PRs we have open to generate the file.  Then just add a `ref: ${{ inputs.branch }}` on line 45 and we'll check out the right branch